assets/xml/objects/object_shopnuts.xml

```diff
@@ -1,26 +1,26 @@
 <Root>
     <File Name="object_shopnuts" Segment="6">
         <!-- Business Scrub Skeleton -->
-        <Skeleton Name="gBusinessScrubSkel" Type="Flex" LimbType="Standard" LimbNone="BIZ_SCRUB_LIMB_NONE" LimbMax="BIZ_SCRUB_LIMB_MAX" EnumName="BizScrubLimb" Offset="0x41A8"/>
+        <Skeleton Name="gBusinessScrubSkel" Type="Flex" LimbType="Standard" LimbNone="BUSINESS_SCRUB_LIMB_NONE" LimbMax="BUSINESS_SCRUB_LIMB_MAX" EnumName="BUSINESSScrubLimb" Offset="0x41A8"/>
```

Comment:
```suggestion
        <Skeleton Name="gBusinessScrubSkel" Type="Flex" LimbType="Standard" LimbNone="BUSINESS_SCRUB_LIMB_NONE" LimbMax="BUSINESS_SCRUB_LIMB_MAX" EnumName="BusinessScrubLimb" Offset="0x41A8"/>
```

Unless Dragorn wants it named EnDns for now

Reply:
:shrug: 
(also I'm not deciding for everyone and if I give the impression I want to or am, please confront me about it)

Reply:
@Dragorn421 nah nothing like that, just wasn't 100% on the current naming conventions for object doc and just guessed you might know more/figured you had an eye on this pr.